packages/ui/src/vuestic-plugin/vuestic-plugin-without-components.ts

```diff
@@ -1,6 +1,6 @@
 import type { GlobalConfig } from '../main'
 import type { App } from 'vue'
-import { DropdownPopperSubplugin, ToastInstall, GlobalConfigPlugin, ColorHelpersPlugin } from './vuestic-plugins'
+import { DropdownPopperSubplugin, ToastInstall, GlobalConfigPlugin, ColorHelpersPlugin, ClassHelpersPlugin } from './vuestic-plugins'
 
 export const VuesticPluginsWithoutComponents = {
   install (app: App, vuesticConfig: GlobalConfig): void {
```

Comment:
If you made color as required prop, make sure we don't force user to provide whole config in plugin and `mergeGlobalConfig`.  (`setGlobalConfig` likely must request all required fields.)

```suggestion
  install (app: App, vuesticConfig: Partial<GlobalConfig>): void {
```